### DISCUSSION TITLE
Baumann Koelliker Gruppe

### DISCUSSION THREAD

**Discussion by aaronmeder** [flyntwp/flynt#459] [Show and tell]
We had the opportunity to help design and build a set of websites for a group of swiss electrician companies called "Baumann Koelliker". A part from the website for the [group](https://baumann-koelliker-gruppe.ch/) we've set up a separate [job platform](https://jobs.baumann-koelliker-gruppe.ch/) and 16 websites for the individual companies that are part of the group.

![bkg-multi-desktop-clay-mockup](https://user-images.githubusercontent.com/5564163/189058289-9ceff7dd-92cb-404a-8a2d-b8f152380b09.jpg)

Flynt with it's component / Timber based approach was a perfect toolset that helped us to maximize reusing common parts and let to a flexible yet easy-to-understand Wordpress admin experience.

The customer actually didn't believe we're going to finish on time 😄

All the websites:
-  [Baumann Koelliker Gruppe](https://www.baumann-koelliker-gruppe.ch/)
-  [Jobs Platform](https://jobs.baumann-koelliker-gruppe.ch/)
-  [Baumann Koelliker AG](https://www.baumann-koelliker.ch/)
-  [Beltech Schaltanlagen AG](https://www.beltech.ch/)
-  [Brupbacher Gatti AG](https://www.brupbacher-gatti.ch/)
-  [Bütler Elektro Telecom AG](https://www.buetler-elektro.ch/)
-  [De Boni Elektro AG](https://www.debonielektro.ch/)
-  [Electro Inspect AG](https://www.electroinspect.ch/)
-  [Elektro M+C AG](https://www.elektro-mc.ch/)
-  [Ellenbrok Hugentobler AG](https://www.elhag.ch/)
-  [Glatttal Installationen AG](https://www.glattal-elektro.ch/)
-  [Haupt+Studer AG](https://www.haupt-studer.ch/)
-  [Käser Elektro AG](https://www.kaeser-elektro.ch/)
-  [Miesch Elektro AG](https://www.miesch-elektro.ch/)
-  [Netfon Solutions AG](https://netfon-solutions.ch/)
-  [Ruther AG](https://www.ruther.ch/)
-  [Stadler AG](https://www.stadler-elektro.ch/)
-  [Wenger + Wirz AG](https://www.wenger-wirz.ch/)

**Comment by steffenbew** [comment]
Hey @aaronmeder, those websites look fabulous! Quality design, excellent frontend execution and super smooth micro animations. 
I couldn't be happier that those sites were built on top of Flynt.
Congratulations for making it happen and thanks for sharing!
